skimage/measure/profile.py

```diff
@@ -109,3 +124,229 @@ def _line_profile_coordinates(src, dst, linewidth=1):
+    """Return an array of angles that will be used to rotate a point 360 degrees around a line.
+    The number of angles is dependent on the distance between the point and the line.
+    The farther the point from the line, the smaller the angle incrementation.
+
+    Parameters
+    ----------
+    src : 3-tuple of numeric scalar (float or int)
+        A first point where the line is passing through
+    dst : 3-tuple of numeric scalar (float or int)
+        A second point where the line is passing through
+    point : 3-tuple of numeric scalar (float or int)
+        The point to find the distance.
+
+    Returns
+    -------
+    angles : tuple, float
+        The angles that will be used to rotate the sample point to create more sample points around the line.
+    """
+    distance_point_line = _distance_point_line_3d(point, src, dst)
+    if distance_point_line == 0:
+        rotation_angles = np.array([0])
+    else:
+        rotation_angles = np.linspace(0, 2 * constants.pi, 2 * distance_point_line + 3)
+        rotation_angles = np.delete(rotation_angles, len(rotation_angles) - 1)
+    return rotation_angles
+
+
+def _distance_point_line_3d(point, src, dst):
+    """Return the distance between a line and a point in 3d.
+
+    Parameters
+    ----------
+    point : 3-tuple of numeric scalar (float or int)
+        The point to find the distance.
+    src : 3-tuple of numeric scalar (float or int)
+        A first point where the line is passing through
+    dst : 3-tuple of numeric scalar (float or int)
+        A second point where the line is passing through
+
+    Returns
+    -------
+    point : float
+        The distance between the point and the line in units
+    """
+    line_vector = np.subtract(dst, src)
+    length_vector = np.linalg.norm(line_vector)
+    vector1 = np.subtract(point, dst)
+    vector2 = np.subtract(point, src)
+    distance = np.linalg.norm(np.cross(vector1, vector2) / length_vector)
+    return np.abs(distance)
+
+
+def _rotate_point_around_line(point_to_rotate, point_on_line, unit_direction_vector, angle_in_radians):
+    """Return a 3d point that is rotated at an angle of theta around a line
+    passing through a selected point.
+
+    Parameters
+    ----------
+    point_to_rotate : 3-tuple of numeric scalar (float or int)
+        The point to rotate.
+    point_on_line : 3-tuple of numeric scalar (float or int)
+        A point where the line is passing through
+    unit_direction_vector : 3-tuple of numeric scalar (float or int)
+        The unit direction vector of the line
+    angle_in_radians : float or int
+        The angle of rotation in radians
+
+    Returns
+    -------
+    return_value : array, shape (3), float
+        The coordinates of the rotated point around the line
+    """
+    c, b, a = point_on_line
+    z, y, x = point_to_rotate
+    w, v, u = unit_direction_vector
+
+    p1 = (a * (v ** 2 + w ** 2) - u * (b * v + c * w - u * x - v * y - w * z)) * \
```

Comment:
PEP8 actually suggests breaking spacing to indicate operator priority. In the case of formulas like this one, this would be more readable, I think:

```
p1 = (a*(v**2 + w**2) - u*(b*v + c*w - u*x - v*y - w*z)) *
```